client/src/app/interfaces/user.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
```

Comment:
See my comment on the article interface. This could be a change made in a future PR, but I think one of the advantages of typescript is that you can share the types to check that your code is sending and receiving the right object types. If you recreate types in multiple places, you might inadvertently make different types for the same thing.

Reply:
I'll make this change in another PR, like you suggested on Slack. Thanks!